### PR TITLE
Update after introducing Email plugins

### PR DESCRIPTION
**Pull request by korycins**
Changes related to this PR 
https://github.com/mirumee/saleor/pull/6301

### CODE REVIEW

File: docs/developer/available-plugins/user-emails.mdx [hunk 1]
```diff
@@ -13,51 +13,77 @@ The payload is available directly from the email template.
...
-- The invoice for the order is ready (_invoice_ready_)
-- The order has been created (_order_confirmation_)
-- The order has been confirmed by the staff user (_order_confirmed_)
-- The order has been fulfilled (_order_fulfillment_confirmation_)
-- The fulfillment has been updated (_order_fulfillment_update_)
-- The payment has been confirmed (_order_payment_confirmation_)
-- The order has been canceled (_order_canceled_)
-- The order has been refunded (_order_refund_confirmation_)
-
+[WebhookEventTypeEnum](developer/api-reference/enums/webhook-event-type-enum.mdx) contain all availabe events.
```
Comment (korycins) [inline docs/developer/available-plugins/user-emails.mdx:16]: It is a link to Webhook events. Which are not really related to the notification events.

File: docs/developer/available-plugins/user-emails.mdx [hunk 2]
```diff
@@ -13,51 +13,77 @@ The payload is available directly from the email template.
...
 
-- `format_address(address, include_phone=True, inline=False, latin=False)` - converts json address structure to string.
+### format_address
+Applies formatting to the address.
+
+Arguments:
+- **address** (json) - Adress instance in json format.
+- **include_phone** (boolean) - Controls if formatted adress will contain phone number. Default: True
+- **inline** (boolean) - Controlls if address should be formated inline. Default: False
+- **latin** (boolean) - ???
```
Comment (korycins) [inline docs/developer/available-plugins/user-emails.mdx:33]: It is a value passed to i18naddress library. 
https://github.com/mirumee/google-i18n-address#address-latinization

File: docs/developer/available-plugins/user-emails.mdx [hunk 3]
```diff
@@ -13,51 +13,77 @@ The payload is available directly from the email template.
...
 {{get_product_image_thumbnail 60 product.first_image}}
 ```
-- `compare(val1, compare_operator, val2)` - compares two values. Returns true/false value. Available compare_operators: `==`,`!=`,`<`,`<=`,`>=`,`>`
+
+### compare
+Compares two values. Returns true/false value.
+
+Arguments:
+- **val1** (any) - First value.
+- compare_operator (string) - Available operators: `==`,`!=`,`<`,`<=`,`>=`,`>`.
```
Comment (korycins) [inline docs/developer/available-plugins/user-emails.mdx:83]: **compare_operator** ?

File: docs/developer/available-plugins/admin-emails.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: Admin Emails
+---
+
+`AdminEmails` plugin handles email communication with staff (Dashboard users).
+
+Internally the plugin is familiar to [`UserEmails`](developer/available-plugins/user-emails.mdx), with a few notable differences:
```
Comment (karolkielecki) [inline docs/developer/available-plugins/admin-emails.mdx:7]:
```suggestion
Internally, the plugin is similar to [`UserEmails`](developer/available-plugins/user-emails.mdx), with a few notable differences:
```

File: docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+title: Creating and installing Apps from the command line
+sidebar_label: Installing Apps
+---
+
+Saleor provides commands that can be used to create/install an App from the command line.
+
+## Creating local Apps
+
+To create a local App with explicitly provided permissions use the `create_app` command.
```
Comment (karolkielecki) [inline docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx:10]:
```suggestion
To create a local App with explicitly provided permissions, use the `create_app` command.
```

File: docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+If the target URL is not specified, the command will output an API auth token in the following format:
+
+```shell
+{"auth_token": "new-app-token"}
+```
+
+## Installing third-party Apps
+
+To install a `third-party` App based on its manifest URL use the `install_app` command.
```
Comment (karolkielecki) [inline docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx:49]:
```suggestion
To install a `third-party` App based on its manifest URL, use the `install_app` command.
```

File: docs/developer/api-reference/enums/webhook-event-type-enum.mdx [hunk 7]
```diff
@@ -38,48 +39,100 @@ enum WebhookEventTypeEnum {
...
 
+All the events.
+
 #### `ORDER_CREATED`
 
+A new order is placed.
+
 #### `ORDER_CONFIRMED`
 
+An order is confirmed (status change unconfirmed -> unfulfilled) by staff user using OrderConfirm mutation. Also triggers when user finish checkout and shop setting `automatically_confirm_all_new_orders` is enabled.
```
Comment (karolkielecki) [inline docs/developer/api-reference/enums/webhook-event-type-enum.mdx:50]:
```suggestion
An order is confirmed (status change: unconfirmed -> unfulfilled) by a staff user using the OrderConfirm mutation. It also triggers when the user completes the checkout process and the shop setting `automatically_confirm_all_new_orders` is enabled.
```

File: docs/developer/api-reference/enums/webhook-event-type-enum.mdx [hunk 8]
```diff
@@ -38,48 +39,100 @@ enum WebhookEventTypeEnum {
...
 
+A product variant is deleted.
+
 #### `CHECKOUT_CREATED`
 
+A new checkout is created.
+
 #### `CHECKOUT_UPDATED`
 
+A checkout is updated. Also triggers for all updates related to a checkout.
```
Comment (karolkielecki) [inline docs/developer/api-reference/enums/webhook-event-type-enum.mdx:118]:
```suggestion
A checkout is updated. It also triggers all updates related to the checkout process.
```

File: docs/developer/available-plugins/admin-emails.mdx (hunk 4 above)
Comment (karolkielecki) [inline docs/developer/available-plugins/admin-emails.mdx]:
```suggestion
Internally, the plugin is similar to [`UserEmails`](developer/available-plugins/user-emails.mdx), with a few notable differences:
```

File: docs/developer/api-reference/enums/webhook-event-type-enum.mdx (hunk 8 above)
Comment (karolkielecki) [inline docs/developer/api-reference/enums/webhook-event-type-enum.mdx:118]:
```suggestion
A checkout is updated. It also triggers all updates related to the checkout process.
```

File: docs/developer/api-reference/enums/webhook-event-type-enum.mdx (hunk 7 above)
Comment (karolkielecki) [inline docs/developer/api-reference/enums/webhook-event-type-enum.mdx:50]:
```suggestion
An order is confirmed (status change: unconfirmed -> unfulfilled) by a staff user using the OrderConfirm mutation. It also triggers when the user completes the checkout process and the shop setting `automatically_confirm_all_new_orders` is enabled.
```

File: docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx (hunk 5 above)
Comment (karolkielecki) [inline docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx:10]:
```suggestion
To create a local App with explicitly provided permissions, use the `create_app` command.
```

File: docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx (hunk 6 above)
Comment (karolkielecki) [inline docs/developer/extending/apps/creating-and-installing-apps-from-the-command-line.mdx]:
```suggestion
To install a `third-party` App based on its manifest URL, use the `install_app` command.
```

**Review by karolkielecki — CHANGES_REQUESTED** [review]

**Pull request merged**